frontendGraph/src/main/java/com/virtuslab/gitmachete/frontend/graph/print/elements/EdgePrintElement.java

```diff
@@ -0,0 +1,67 @@
+
+package com.virtuslab.gitmachete.frontend.graph.print.elements;
+
+import com.intellij.vcs.log.graph.api.elements.GraphEdge;
+import lombok.Getter;
+import org.checkerframework.checker.index.qual.NonNegative;
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+import com.virtuslab.gitmachete.frontend.graph.GraphElementManager;
+
+@Getter
+public class EdgePrintElement extends PrintElementWithGraphElement implements IEdgePrintElement {
+
+  private final Type type;
+
+  @NonNegative
+  private final int positionInOtherRow;
+
+  public EdgePrintElement(@NonNegative int rowIndex,
+      @NonNegative int positionInCurrentRow,
+      @NonNegative int positionInOtherRow,
+      Type type,
+      GraphEdge graphEdge,
+      GraphElementManager graphElementManager) {
+    super(rowIndex, positionInCurrentRow, graphEdge, graphElementManager);
+    assert (type != Type.RIGHT && positionInCurrentRow == positionInOtherRow)
```

Comment:
For clarity: if `type == Type.RIGHT`, then always `positionInOtherRow == positionInCurrentRow + 1`? or can it as well be `positionInOtherRow == positionInCurrentRow`?

Reply:
good point, we can ignore the `positionInOtherRow` since all edges are orthogonal

Reply:
done